network-api/networkapi/buyersguide/pagemodels/product_category.py

```diff
@@ -3,7 +3,6 @@
 
 from wagtail.snippets.models import register_snippet
 
-from .products.base import Product
```

Comment:
let's put this back in for now
```suggestion
from .products.base import Product
```